_topic_map.yml

```diff
@@ -1335,12 +1335,12 @@ Topics:
   Topics:
   - Name: Getting started with Helm on OpenShift Container Platform
     File: getting-started-with-helm-on-openshift-container-platform
-#- Name: tkn CLI
-#  Dir: tkn_cli
-#  Distros: openshift-enterprise,openshift-webscale,openshift-origin
-#  Topics:
-#  - Name: Installing tkn
-#    File: installing-tkn
+- Name: tkn CLI
```

Comment:
Suggest naming this "OpenShift Pipelines CLI (tkn)". It's good to give the most descriptive name at the top level so people can find it easier (especially if they don't know what it is), and it's consistent with what we do for the others, for example: "OpenShift Do developer CLI (odo)".

Reply:
I agree with you @bergerhoffer . I have already made this change as part of https://github.com/openshift/openshift-docs/pull/20692/files#diff-9ff4392b3b4341f06984a7cd81583cd5

I did not change this file here as I had to comment it back. Had uncommented it for generating preview. Will comment it back once you are ok with the changes to be merged. 

We will be raising a separate PR for just _topics.yaml file once all the content for pipelines is merged. 

Let me know if this sounds ok to you.

Thanks!1

Reply:
Hi @nkakkar81, yes that sounds fine to me to have the name change be handled by #20692. Thanks!